source/scss/components/_footer-hotline.scss

```diff
@@ -2,8 +2,8 @@
 
 // TODO font-size and padding variables should go here, speak to your designer, to find out, if we use the sizes in a more global context
 .footer-hotline {
-  background-color: $secondary;  // $secondary -> dark gray
-  color: $white;
+  background-color: $gray-100;
+  color: $gray-600;
```

Comment:
I wonder to which desk I should go to talk about that. :)

Reply:
Let me discuss with René this point. Because this should be specified from design-side already. But we don't have a design-system yet. 
Designers need to decide if this specific grey always bounds elements to be coloured in secondary, or if some of them always stay grey. Because grey is a neutral color and has a special place in designs next to accent colours, like the primary color. :)

Reply:
I checked with Eje and the designers and found out that the secondary is not  a grey. It is a blue according to the company's branding. So I asked the designer to take a look at the styleguide itself and hopefully there will be a refactoring soon.
For the grey itself, Eje said, we should stick to the $grey-600 until then.